wayfire-shadows| Run-time dependency wlroots found: NO (tried pkgconfig and cmake)

subprojects/wayfire-shadows/meson.build:17:11: ERROR: Dependency "wlroots" not found, tried pkgconfig and cmake

windecor| Run-time dependency wlroots found: NO (tried pkgconfig and cmake)

subprojects/windecor/meson.build:18:11: ERROR: Dependency "wlroots" not found, tried pkgconfig and cmake

--- subprojects/wayfire-shadows/meson.build.orig	2023-09-08 22:38:12 UTC
+++ subprojects/wayfire-shadows/meson.build
@@ -14,7 +14,7 @@ wayfire  = dependency('wayfire')
 )
 
 wayfire  = dependency('wayfire')
-wlroots  = dependency('wlroots')
+wlroots  = dependency('wlroots-0.17')
 wfconfig = dependency('wf-config')
 
 add_project_arguments(['-DWLR_USE_UNSTABLE'], language: ['cpp', 'c'])
--- subprojects/windecor/meson.build.orig	2023-09-30 05:08:04 UTC
+++ subprojects/windecor/meson.build
@@ -15,7 +15,7 @@ pixman   = dependency('pixman-1')
 
 wayfire  = dependency('wayfire')
 pixman   = dependency('pixman-1')
-wlroots  = dependency('wlroots')
+wlroots  = dependency('wlroots-0.17')
 wfconfig = dependency('wf-config')
 cairo    = dependency('cairo')
 rsvg     = dependency( 'librsvg-2.0' )
